readme_renderer/__about__.py

```diff
@@ -23,10 +23,10 @@
                'descriptions for Warehouse')
 __uri__ = "https://github.com/pypa/readme_renderer"
 
-__version__ = "18.1"
+__version__ = "19.0"
 
-__author__ = "Donald Stufft"
-__email__ = "donald@stufft.io"
+__author__ = "The Python Packaging Authority"
+__email__ = "admin@mail.pypi.org"
```

Comment:
This should probably distutils-sig or something.

Reply:
I would like for it to be an address that is robustly monitored, and that doesn't require the user to sign up for something (such as a list) in order to contact the owner...